setup.py

```diff
@@ -37,6 +37,7 @@
         "plyvel==1.0.4",
         "coincurve>=7.0.0,<8.0.0",
         "web3>=4.1.0,<5.0.0",
+        "upnpclient>=0.0.8",
```

Comment:
Maybe this belongs in the `p2p` extra, instead?

Reply:
Yeah, it does!